mindsdb/integrations/libs/handler_helpers.py

```diff
@@ -29,3 +29,20 @@ def define_handler(_type):
     if _type == 'mariadb':
         return MariaDBHandler
     return None
+
+
+def _clean_pdef(pdef):
+    """ Simple helper method that removes backticks from columns that collide with special string in mindsdb_sql. """
+    if 'target' in pdef and '`target`' == pdef['target']:
+        pdef['target'] = pdef['target'].replace('`', '')
```

Comment:
I think string replace is not safe. For example in case field really contents '`'
It is better to remove backticks in place where they appears in model_controller.py by replacing: 
```
'order_by': str(getattr(statement, 'order_by')[0])
...
problem_definition['timeseries_settings']['group_by'] = [str(col) for col in statement.group_by]
```
to:
```
'order_by': getattr(statement, 'order_by')[0].field.parts[-1]
...
problem_definition['timeseries_settings']['group_by'] = [col.parts[-1] for col in statement.group_by]
```